openreview/conference/templates/reviewerWebfield.js

```diff
@@ -283,10 +283,15 @@ var displayPaperRanking = function(notes, paperRankingInvitation, paperRankingTa
   var invitation = paperRankingInvitation ? _.cloneDeep(paperRankingInvitation) : null;
   if (invitation) {
     var availableOptions = ['No Ranking'];
+    var validTags = [];
     notes.forEach(function(note, index) {
       availableOptions.push((index + 1) + ' of ' + notes.length );
+      noteTags = paperRankingTags.filter(function(tag) { return tag.forum == note.forum; })
+      if (noteTags.length) {
+        validTags.push(noteTags[0]);
+      }
```

Comment:
```suggestion
      var noteTag = paperRankingTags.find(function(tag) { return tag.forum === note.forum; })
      if (noteTag) {
        validTags.push(noteTag);
      }
```

Reply:
I'm keeping the filter here because I would need to check which is the most recent tag to use and ignore the others, if there are duplicate tags for the same paper.

Reply:
I see, makes sense. Changing the == to === is the only change then.

Reply:
done